modules/setting/setting.go

```diff
@@ -33,6 +33,9 @@ const (
 	HTTP  Scheme = "http"
 	HTTPS Scheme = "https"
 	FCGI  Scheme = "fcgi"
+
+	RUN_MODE_PROD = "prod"
+	RUN_MODE_TEST = "test"
 )
```

Comment:
I leave it out because it wouldn't be used by now. But I think I could add it for maybe future use.